Cannot install on NODE 20 LTS

### Describe the Improvement

Currently it seems there is a dependency which prevents beeing able to install directus on node LTS 20.x

```
ERR_PNPM_UNSUPPORTED_ENGINE  Unsupported environment (bad pnpm and/or Node.js version)

This error happened while installing the dependencies of directus@10.7.2
 at @directus/api@14.0.2
 at tedious@16.1.0
 at @azure/identity@2.1.0

Your Node version is incompatible with "registry.npmjs.org/@azure/msal-node/1.18.4".

Expected version: 10 || 12 || 14 || 16 || 18
Got: v20.9.0
```

Indeed latest @azure/msal-node does support node 20, which evidently tedious need to update.

update: there is  PR for this. creating this ticket anyway to make sure that directus also updates tedious.
https://github.com/tediousjs/tedious/issues/1585

--- a/.changeset/grumpy-planets-wait.md
+++ b/.changeset/grumpy-planets-wait.md
@@ -1,0 +1,5 @@
+---
+'@directus/api': patch
+---
+
+Updated Tedious dependency

--- a/api/package.json
+++ b/api/package.json
@@ -224,7 +224,7 @@
 		"nodemailer-sendgrid": "1.0.3",
 		"pg": "8.11.0",
 		"sqlite3": "5.1.6",
-		"tedious": "16.1.0"
+		"tedious": "16.6.1"
 	},
 	"engines": {
 		"node": ">=18.0.0"

